### PR TITLE
New year 

### PR DESCRIPTION
**Pull request by Sentient07**
Happy new year! :)

### CODE REVIEW

**Comment by nouiz** [comment]
thanks. if I grep for 2015 in Theano I find others. Do you want to also update them?

**Comment by Sentient07** [comment]
@nouiz Yeah, sure! Actually, i was about to do that today.

**Comment by Sentient07** [comment]
@nouiz I've ```grep```'ed all 2015 and changed the necessary.

File: doc/LICENSE.txt [hunk 1]
```diff
@@ -3,14 +3,14 @@
...
-Copyright (c) 2008--2015, Theano Development Team
+Copyright (c) 2008--2016, Theano Development Team
 All rights reserved.
 
 Contains code from NumPy, Copyright (c) 2005-2011, NumPy Developers.
 All rights reserved.
 
 Contain CnMeM under the same license with this copyright:
-Copyright (c) 2015, NVIDIA CORPORATION. All rights reserved.
+Copyright (c) 2016, NVIDIA CORPORATION. All rights reserved.
```
Comment (abergeron) [inline doc/LICENSE.txt]: There is still no new code for this, so the copyright year remains 2015.  We should update it when we next do a sync.

File: theano/compat/six.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
 """Utilities for writing code that runs on Python 2 and 3"""
 
-# Copyright (c) 2010-2015 Benjamin Peterson
+# Copyright (c) 2010-2016 Benjamin Peterson
```
Comment (abergeron) [inline theano/compat/six.py]: This isn't our code so the year should only change when (if) we import a new version of six.

File: theano/sandbox/cuda/cnmem.cpp [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ///////////////////////////////////////////////////////////////////////////////////////////////////
-// Copyright (c) 2015, NVIDIA CORPORATION. All rights reserved.
+// Copyright (c) 2016, NVIDIA CORPORATION. All rights reserved.
```
Comment (abergeron) [inline theano/sandbox/cuda/cnmem.cpp]: Not our code again

File: theano/sandbox/cuda/cnmem.h [hunk 4]
```diff
@@ -1,5 +1,5 @@
 /* ********************************************************************** 
- * Copyright (c) 2015, NVIDIA CORPORATION. All rights reserved.
+ * Copyright (c) 2016, NVIDIA CORPORATION. All rights reserved.
```
Comment (abergeron) [inline theano/sandbox/cuda/cnmem.h]: Idem

File: theano/tests/test_flake8.py [hunk 5]
```diff
@@ -14,7 +14,7 @@
     flake8_available = False
 
 __authors__ = ("Saizheng Zhang")
-__copyright__ = "(c) 2015, Universite de Montreal"
+__copyright__ = "(c) 2016, Universite de Montreal"
```
Comment (abergeron) [inline theano/tests/test_flake8.py]: I'm not sure about this one ...

Comment (nouiz) [inline theano/tests/test_flake8.py]: About this one, we wanted to do as in this file everywhere in Theano. But
we didn't do it. But as this is our license, I would update it. I'm not
sure if we should keep it, so by not being sure, I would just update it.

File: doc/LICENSE.txt (hunk 1 above)
Comment (Sentient07) [inline doc/LICENSE.txt]: Not sure, but since we use CUDA, i had thought i should update NVIDIA license as well. Will revert

Comment (abergeron) [inline doc/LICENSE.txt]: That copyright is for the CnMeM code that we embed.  CUDA has a separate license.

File: theano/tests/test_flake8.py (hunk 5 above)
Comment (abergeron) [inline theano/tests/test_flake8.py]: Ok then, makes sense to update this.

**Comment by Sentient07** [comment]
@nouiz @abergeron have made the changes

**Comment by abergeron** [comment]
Ok with me (but still waiting for travis to be sure).

**Pull request merged**